lms/startup.py

```diff
@@ -41,6 +41,8 @@ def run():
 
     add_mimetypes()
 
+    microsite.enable_microsites(log)
```

Comment:
Sure. It goes like this, we are doing similar things, but for different components.

We currently use 2 template rendering engines, mako and django_templates, and one of them (django templates), requires the directories be added before the django.setup(). Mako requires the directories to be added after the django setup. That is why we have [this](https://github.com/edx/edx-platform/pull/11224/files#diff-4e3eb53d42974c1129574b282379d600R297) that modified the DIRS for the django_templates setup before the django.setup() and we also have [this](https://github.com/edx/edx-platform/pull/11073/files#diff-4e3eb53d42974c1129574b282379d600R280) which does the same for mako after the django.setup(). I did left [this line](https://github.com/edx/edx-platform/pull/11073/files#diff-4e3eb53d42974c1129574b282379d600R279) even when it was not necessary, just to be sure, but we can remove it and nothing should be affected.